src/index.html

```diff
@@ -5,8 +5,449 @@
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Samsung</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Smartphones</a>
+          </li>
+          <li class="header__nav-item">
+            <a
+              href="#"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Gadgets</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Tablets</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Photo</a>
+          </li>
+          <li class="header__nav-item">
+            <a href="#" class="header__nav-link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <div class="catalog catalog--position">
+        <section class="card card--position" data-qa="card">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="product-item-imac27"
+          >
+
+          <div class="card__description">
+            <h5 class="card__description-title">
+              APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+            </h5>
+
+            <span class="card__description-code">
+              Product code: 195434
+            </span>
+          </div>
+
+          <div class="card__review">
+            <div class="card__stars card__stars--4">
+              <div class="card__stars-star"></div>
+
+              <div class="card__stars-star"></div>
+
+              <div class="card__stars-star"></div>
+
+              <div class="card__stars-star"></div>
+
+              <div class="card__stars-star"></div>
+            </div>
+
+            <div class="card__review-amount">Reviews: 5</div>
+          </div>
+
+          <div class="card__price">
+            <span class="card__price-name">Price:</span>
+
+            <span class="card__price-cost">$2,199</span>
+          </div>
+
+          <div class="card__button" >
+            <a
+              class="card__button-link"
+              href="#"
+              data-qa="card-hover"
+            >
+              buy
+            </a>
+          </div>
+        </section>
+
+        <section class="card card--position">
+          <img
+            class="card__image"
+            src="images/imac.jpeg"
+            alt="product-item-imac27"
+          >
+          <div class="card__description">
+            <span class="card__description-title">
```

Comment:
The card title is incorrectly wrapped in 'span' element. It should be inside heading tags like 'h2' for better SEO and accessibility.